DevoxxClientMobile/src/main/java/com/devoxx/DevoxxDrawerPresenter.java

```diff
@@ -114,7 +114,7 @@ public void postConstruct() {
     }
 
     private void checkAndUpdateItems() {
-        if (service.getConference().getFloorPlans().size() == 0) {
+        if (service.getConference().getFloorPlans() != null && service.getConference().getFloorPlans().size() == 0) {
```

Comment:
Shouldn't this be checking for null instead? So when it's null or empty, remove the exhibition menu item from the drawer?

    if (floorplans == null || floorplans.isEmpty())

Reply:
The null check was added to make sure `size()` is not called on a null object. You might be right about reversing the condition to make it more clear to readers.